hack/clean/clean.go

```diff
@@ -105,7 +105,9 @@ func run(ctx context.Context, log *logrus.Entry) error {
 		// TODO: Remove when people stop re-using shared RG for their vnets
 		// and tags are not removed.
 		if strings.HasPrefix(*resourceGroup.Name, "aro-v4-shared") ||
-			strings.HasPrefix(*resourceGroup.Name, "v4-") {
+			strings.HasPrefix(*resourceGroup.Name, "v4-eastus") ||
+			strings.HasPrefix(*resourceGroup.Name, "v4-australiasoutheast") ||
+			strings.HasPrefix(*resourceGroup.Name, "v4-westeurope") {
```

Comment:
You say "we need to match specific names". Should we get rid of `strings.HasPrefix` in that case?

Reply:
We need to refactor all this part :/ and fix how we do this. I just needed something quick and dirty to fix now. If I get to this today I will fix this :/

Reply:
https://github.com/Azure/ARO-RP/issues/927